frontend/components/PaginatedList/PaginatedList.tsx

```diff
@@ -0,0 +1,225 @@
+import React, {
+  useState,
+  useEffect,
+  useImperativeHandle,
+  forwardRef,
+  Ref,
+} from "react";
+import classnames from "classnames";
+import { ReactElement } from "react-markdown/lib/react-markdown";
+import Checkbox from "components/forms/fields/Checkbox";
+import Spinner from "components/Spinner";
+import TooltipTruncatedText from "components/TooltipTruncatedText";
+// @ts-ignore
+import Pagination from "components/Pagination";
```

Comment:
Don't hate the messenger >.<

Just noticed this on `components/Pagination`:

```
/**
 * WARNING: DEPRICATED:
 * This pagination component is DEPRICATED. It is being kept around until we replace its
 * use. For now use the Pagination component in the pages/ManageControlsPage/components.
 */
```

Is there a way we can move to the other pagination?

Reply:
Ok, looking into it.